manual_flagging/flagging.py

```diff
@@ -284,10 +284,10 @@ def create_bins_and_labels(input_list):
 
 if og_mansueto_tris:
     # Filter by triad code and market type
-    og_mansueto_filter = ~df["triad_code"].isin(og_mansueto_tris)
+    og_mansueto_filter = df["triad_code"].astype(int).isin(og_mansueto_tris)
 
-    df_res_og_mansueto = df[(df["indicator"] == "res") & triad_code_filter]
-    df_condo_og_mansueto = df[(df["indicator"] == "condo") & triad_code_filter]
+    df_res_og_mansueto = df[(df["indicator"] == "res") & og_mansueto_filter]
+    df_condo_og_mansueto = df[(df["indicator"] == "condo") & og_mansueto_filter]
```

Comment:
Not sure how I missed this, but the filter operation was incorrect here, resulting in incorrect sales being flagged. This is fixed and tested now.